test/e2e_node/remote/remote.go

```diff
@@ -295,22 +294,32 @@ func RunRemote(archive string, host string, cleanup bool, junitFilePrefix string
 	}
 
 	glog.Infof("Copying test artifacts from %s", host)
-	scpErr := getTestArtifacts(host, tmp)
+	scpErr := getTestArtifacts(host, workspace)
 	if scpErr != nil {
 		aggErrs = append(aggErrs, scpErr)
 	}
 
 	return output, len(aggErrs) == 0, utilerrors.NewAggregate(aggErrs)
 }
 
+func getTimestamp() string {
+	t := time.Now()
+	return fmt.Sprintf("%d-%02d-%02dT%02d-%02d-%02d", t.Year(), t.Month(), t.Day(),
+		t.Hour(), t.Minute(), t.Second())
```

Comment:
I want to avoid using `:` in file name, that's why I format the timestamp myself. :)